src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -2,12 +2,11 @@
   "Logging": {
     "IncludeScopes": false,
     "LogLevel": {
-      "Default": "Error",
```

Comment:
Information is too verbose by default, and we have Console logging enabled, `Warning` would be fine to me.